db/migrate/20160114153948_delete_cloud_network_id_from_network_ports.rb

```diff
@@ -1,6 +1,11 @@
 class DeleteCloudNetworkIdFromNetworkPorts < ActiveRecord::Migration
-  def change
+  def up
     remove_index :network_ports, :column => :cloud_network_id
```

Comment:
@Ladas Re: [comment](https://github.com/ManageIQ/manageiq/pull/6899#issuecomment-187817847) Definitely; the remove_index comment is just that you could remove the line here ^_^ Still have to readd the index on `down`, right :+1: